addressfield_geosuggest.module

```diff
@@ -8,16 +8,27 @@
 /**
  * Returns the set google places API.
  *
- * The key is set via the variable "addressfield_geosuggest_api_key".
+ * The key is set via the variable "addressfield_geosuggest_api_key" and
+ * "addressfield_geosuggest_api_key_server".
+ *
+ * @param string $type
+ *   Type of key to return, default is browser.
  *
  * @return null|string
  *   They key as a string if set, otherwise null.
  */
-function addressfield_geosuggest_api_key() {
-  $api_key = variable_get('addressfield_geosuggest_api_key', NULL);
+function addressfield_geosuggest_api_key($type = 'browser') {
+  $api_key = NULL;
+  if ($type == 'browser') {
+    $api_key = variable_get('addressfield_geosuggest_api_key', NULL);
+  }
+  elseif ($type == 'server') {
+    $api_key = variable_get('addressfield_geosuggest_api_key_server', NULL);
```

Comment:
We should remove the new variable if the module is uninstalled.

Reply:
yes, was already done here:
https://github.com/jobiqo/addressfield_geosuggest/pull/10/files#diff-b47e5a908b8c86f471a7c8892420460bR8